### ISSUE TITLE
Rocket Chat 3.5.4 Windows client not showing the interface or icon in the system tray

### ISSUE THREAD

**Issue by Tonysui** [RocketChat/Rocket.Chat.Electron#2174]
**Describe the bug**
The Windows client interface not showing up and there is no icon in the system tray when I started the app.

What operating system and which version? Windows 10 Pro 2004
Which version of Rocket.Chat (Server)? 
Which version of Rocket.Chat.Electron (Electron/Desktop)? Rocket Chat 3.5.4 Windows client
Is there any setting relevant changed? Updated the client 

**To Reproduce**
1. Updated Rocket Chat client from 3.2.2 to 3.5.4 by EXE installer
2. Reboot PC
3. Try to open the Rocket Chat application from the desktop shortcut
4. There was no interface showing up or icon in the system try however there were 3 rocket chat process in the task manager
5. Force close the 3 rocket chat process and try to open the app again still not showing the interface

**Expected behavior**
The application interface show up and allow me to log in

**Additional context**
I also deleted the rocket.chat directory from the appdata/roaming and try again however still not showing the RC window.
![YNJq3M42lu](https://user-images.githubusercontent.com/5986842/134114403-1581dd6c-b9d7-4061-8385-6e377ea954cf.png)

**Comment by ankar84** [comment]
Not confirm that issue at my side
![image](https://user-images.githubusercontent.com/4023037/134121142-187aadca-889a-4fdb-ab84-c057adc79061.png)
![image](https://user-images.githubusercontent.com/4023037/134121168-7042d3d7-d544-423d-85ce-551272183c72.png)

**Comment by Tonysui** [comment]
Update: 

I tired everything around then I found when I deleted the servers.json file then the application could be started properly. I reviewed the servers.json file I had a comma at end of the server list as below:

{
  "Demo Rocket Chat": "https://demo.rocket.chat",
  "Open Rocket Chat": "https://open.rocket.chat",
}

When I deleted the comma and put the servers.json back to the app directory, the app could be started properly. 
I used this servers.json file for the verson 3.2.2 and before and haven't got this issue before.
